UI/setup/select_coa.html

```diff
@@ -24,11 +24,19 @@
            label = text('Chart of accounts')
     };
 ELSE;
+    lang = '';
+    IF languages;
+        lang = languages.pop;
+        IF lang.search('-');
+            lang = lang.split('-').1;
+        END;
+    END;
```

Comment:
As an additional layer.
Accept-Language header only viewable server side provides a prioritised list of Lang's preferred by the users browser setup